Core/Debugger/WebSocket/HLESubscriber.cpp

```diff
@@ -480,9 +493,42 @@ void WebSocketHLEFuncScan(DebuggerRequest &req) {
 	if (!req.ParamU32("size", &size))
 		return;
 
+	bool has_recreate = req.HasParam("recreate");
+	bool recreate = false;
+	if (has_recreate) {
+		if (!req.ParamBool("recreate", &recreate)) {
+			return;
+		}
+	}
+
 	if (!Memory::IsValidRange(addr, size))
 		return req.Fail("Address or size outside valid memory");
 
+	if (recreate) {
+		// The following code doesn't do what I wanted it to do AND it's debatable whether it's needed at all
+		// In short, I wanted to shift the start and the end of the range before removal
+
+		//// let's see if the last function is partially inside our range
```

Comment:
Is there then any reason to keep this commented-out block of code?

Reply:
I don't mind it being cut.